src/templates/podcast.tsx

```diff
@@ -0,0 +1,219 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import ArticleSchema from '~components/dev-hub/article-schema';
+import Layout from '~components/dev-hub/layout';
+import PodcastJumbotron from '~components/dev-hub/podcast-jumbotron';
+import SEO from '~components/dev-hub/SEO';
+import styled from '@emotion/styled';
+import { lineHeight, screenSize, size } from '~components/dev-hub/theme';
+import { P } from '~components/dev-hub/text';
+import AudioPlayer from '~components/dev-hub/podcast-player/audio-player';
+import BlogShareLinks from '~components/dev-hub/blog-share-links';
+import ShareFooter from '~components/dev-hub/article-share-footer';
+import { useSiteMetadata } from '~hooks/use-site-metadata';
+import { addTrailingSlashIfMissing } from '~utils/add-trailing-slash-if-missing';
+import parse from 'html-react-parser';
+
+
+const PODCAST_BREADCRUMB = [
+    {
+        label: 'Home',
+        target: '/',
+    },
+    { 
+        label: 'Learn', 
+        target: '/learn' 
+    },
+];
+
+const TOOLTIP_TEXT = 'Podcast link copied to clipboard!';
+
+const Container = styled('div')`
+    margin: 0 auto;
+
+    @media ${screenSize.largeAndUp} {
+        display: flex;
+        justify-content: center;
+        margin-top: ${size.xxlarge};
+    }
+`;
+
+const Icons = styled('div')`
+    margin: 0 ${size.small};
+
+    span {
+        &:not(:first-of-type) {
+            margin-left: ${size.small};
+        }
+        padding: 0 ${size.tiny};
+    }
+
+    @media ${screenSize.largeAndUp} {
+        display: flex;
+        flex-direction: column;
+        margin: 0 ${size.default};
+
+        span:not(:first-of-type) {
+            margin-top: ${size.small};
+        }
+    }
+`;
+
+const Content = styled('article')`
+    margin: 0 auto;
+    max-width: ${size.maxContentWidth};
+    padding-left: ${size.small};
+    padding-right: ${size.small};
+    width: 100%;
+
+    @media ${screenSize.largeAndUp} {
+        margin: initial;
+    }
+`;
+
+const StyledParagraph = styled('p')`
+    line-height: ${lineHeight.default};
+    a {
+           color: white;
+      }
+`;
+
+const StyledPlayer = styled(AudioPlayer)`
+    margin-bottom: ${size.mediumLarge};
+
+    @media ${screenSize.largeAndUp} {
+        margin-bottom: ${size.large};
+    }
+`;
+
+const StyledBlogShareLinks = styled(BlogShareLinks)`
+    flex-direction: column;
+    align-items: center;
+    @media ${screenSize.largeAndUp} {
+        > * {
+            &:not(:first-of-type) {
+                margin-top: ${size.medium};
+            }
+        }
```

Comment:
When I click the link to copy, I notice the component shifts either down or to the right depending on which one it is. Do you have any idea as to which css might be causing this?